sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -215,3 +224,7 @@ def test_sparse_coder_estimator():
                        transform_alpha=0.001).transform(X)
     assert_true(not np.all(code == 0))
     assert_less(np.sqrt(np.sum((np.dot(code, V) - X) ** 2)), 0.1)
+
+if __name__ == '__main__':
+    import nose
+    nose.runmodule()
```

Comment:
Please do not add such boilerplate. Use nosetests `sklearn/decomposition/tests/test_dict_learning.py` to run the tests of a specific test module.